tests/testthat/test_binning.R

```diff
@@ -48,6 +48,16 @@ test_that("bin works", {
     expect_true(is.numeric(res_2))
     expect_equal(res$x, res_2)
 
+    ## Ensure y order does not matter
```

Comment:
I know it is done above as well but why we need such a "complex" unit test if I could simply test `bin(1:3, 1:3)` vs `bin(3:1, 3:1)`.

Unit tests should be as easy as possible and should require nearly no runtime (I know we are not good at both in all RforMassSpectrometry packages).